addons/external_intercom/functions/fnc_getInteractionPoint.sqf

```diff
@@ -0,0 +1,33 @@
+params ["_vehicle"];
+
+private _customPosition = getArray(configOf _vehicle);
```

Comment:
does this make sense?
How can you call getArray on a vehicle class??

Reply:
Whoops